<ui version="4.0" >
 <comment>*********************************************************************
**
** Copyright (C) 1992-2008 Trolltech ASA. All rights reserved.
**
** This file is part of the Qt Designer of the Qt Toolkit.
**
** This file may be used under the terms of the GNU General Public
** License versions 2.0 or 3.0 as published by the Free Software
** Foundation and appearing in the files LICENSE.GPL2 and LICENSE.GPL3
** included in the packaging of this file.  Alternatively you may (at
** your option) use any later version of the GNU General Public
** License if such license has been publicly approved by Trolltech ASA
** (or its successors, if any) and the KDE Free Qt Foundation. In
** addition, as a special exception, Trolltech gives you certain
** additional rights. These rights are described in the Trolltech GPL
** Exception version 1.2, which can be found at
** http://www.trolltech.com/products/qt/gplexception/ and in the file
** GPL_EXCEPTION.txt in this package.
**
** Please review the following information to ensure GNU General
** Public Licensing requirements will be met:
** http://trolltech.com/products/qt/licenses/licensing/opensource/. If
** you are unsure which license is appropriate for your use, please
** review the following information:
** http://trolltech.com/products/qt/licenses/licensing/licensingoverview
** or contact the sales department at sales@trolltech.com.
**
** In addition, as a special exception, Trolltech, as the sole
** copyright holder for Qt Designer, grants users of the Qt/Eclipse
** Integration plug-in the right for the Qt/Eclipse Integration to
** link to functionality provided by Qt Designer and its related
** libraries.
**
** This file is provided "AS IS" with NO WARRANTY OF ANY KIND,
** INCLUDING THE WARRANTIES OF DESIGN, MERCHANTABILITY AND FITNESS FOR
** A PARTICULAR PURPOSE. Trolltech reserves all rights not expressly
** granted herein.
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <class>qdesigner_internal::PaletteEditor</class>
 <widget class="QDialog" name="qdesigner_internal::PaletteEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>409</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>7</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Edit Palette</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="advancedBox" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>Tune Palette</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="1" >
       <widget class="QtColorButton" name="buildButton" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>13</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4" >
       <widget class="QTreeView" name="paletteView" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QRadioButton" name="detailsRadio" >
        <property name="text" >
         <string>Show Details</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="computeRadio" >
        <property name="text" >
         <string>Compute Details</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Quick</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox126" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Preview</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="disabledRadio" >
        <property name="text" >
         <string>Disabled</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QRadioButton" name="inactiveRadio" >
        <property name="text" >
         <string>Inactive</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QRadioButton" name="activeRadio" >
        <property name="text" >
         <string>Active</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" >
       <widget class="qdesigner_internal::PreviewFrame" native="1" name="previewFrame" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>7</hsizetype>
          <vsizetype>7</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtColorButton</class>
   <extends>QToolButton</extends>
   <header>qtcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>qdesigner_internal::PreviewFrame</class>
   <extends>QWidget</extends>
   <header>previewframe.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>qdesigner_internal::PaletteEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>180</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel" >
     <x>134</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>qdesigner_internal::PaletteEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>287</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel" >
     <x>302</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
